doc/manipulating_images/data_preparation.rst

```diff
@@ -214,95 +261,41 @@ Common data preparation steps: resampling, smoothing, filtering
-
-   Specifying a 3x3 matrix that is diagonal as a target_affine fixes the
-   voxel size. For instance to resample to 3x3x3 mm voxels::
-
-    >>> import numpy as np
-    >>> target_affine = np.diag((3, 3, 3))
-
-.. topic:: **Resampling an image to a given reference image**
-
-    Instead of using an affine or shape, it's also possible to resample an
-    image using a reference image with :func:`nilearn.image.resample_to_img`.
-
-.. image:: ../auto_examples/04_manipulating_images/images/sphx_glr_plot_resample_to_template_001.png
-    :target: ../auto_examples/04_manipulating_images/plot_resample_to_template.html
-    :scale: 33%
-
-.. seealso::
-
-   :func:`nilearn.image.resample_img`, :func:`nilearn.image.resample_to_img`
-
-
 Smoothing
 .........
 
-:class:`NiftiMasker` can further be used for local spatial filtering of
-the neuroimaging data to make the data more homogeneous and thus account
-for inter-individual differences in neuroanatomy.
-It is achieved by passing the full-width half maximum (FWHM; in millimeter scale)
-along the x, y, and z image axes by specifying the `smoothing_fwhm` parameter.
-For an isotropic filtering, passing a scalar is also possible.
-The underlying function handles properly the tricky case of non-cubic
-voxels by scaling the given widths appropriately.
+:class:`NiftiMasker` can apply Gaussian spatial smoothing to the
+neuroimaging data, useful to fight noise or for inter-individual
+differences in neuroanatomy. It is achieved by specifying the full-width
+half maximum (FWHM; in millimeter scale) with the `smoothing_fwhm`
+parameter. Anisotropic filtering is also possible by passing 3 scalars
+``(x, y, z)``, the FWHM along the x, y, and z direction.
+
+The underlying function handles properly non-cubic voxels by scaling the
+given widths appropriately.
 
 .. seealso::
 
    :func:`nilearn.image.smooth_img`
 
-
 .. _temporal_filtering:
 
-Temporal Filtering
-..................
+Temporal Filtering and confound removal
+........................................
+
+:class:`NiftiMasker` can also improve aspects of temporal data
+properties, before conversion to voxel signals.
 
-Rather than optimizing spatial properties of the neuroimaging data,
-the user may want to improve aspects of temporal data properties, before
-conversion to voxel signals. :class:`NiftiMasker` can also process voxel signals.
-Here are the possibilities:
+- **Standardization**. Parameter ``standardize``: Signals can be
+  standardized (scaled to unit variance). 
 
-- Confound removal. Two ways of removing confounds are provided. Any linear
-  trend can be removed by activating the `detrend` option.
+- **Frequency filtering**. Low-pass and high-pass filters can be used to
+  remove artifacts. Parameters: ``high_pass`` and ``low_pass``, specified
+  in Hz (note that you must specific the sampling rate in seconds with
+  the ``t_r`` parameter.
+
+- **Confound removal**. Two ways of removing confounds are provided. Any linear
```

Comment:
Make clearer that the first way relies on prespecified behavioral/other indicators while the detrending does not require user-input to perform a notion of confound removal